README.md

```diff
@@ -99,8 +99,8 @@ class SessionsViewModel @Inject constructor(
 
 <img src="arts/viewmodel2.png" width="400px" />
 
-Construct UiModel LiveData from some such LiveData.  
-The `combine` method works like RxJava's combineLatest.  
+Construct UiModel LiveData from some such LiveData.
```

Comment:
This space `  ` is for a line break. What do you think? 👀

Reply:
I think its' BR line break too.  I returned back to spaces.